src/applications/personalization/profile-2/components/connected-apps/ConnectedApps.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import { focusElement } from 'platform/utilities/ui';
+
+import DowntimeNotification, {
+  externalServices,
+} from 'platform/monitoring/DowntimeNotification';
+
+import { ConnectedApps as ConnectedApplications } from 'applications/personalization/profile-2/components/connected-apps/components/ConnectedApps';
+
+import { handleDowntimeForSection } from 'applications/personalization/profile360/components/DowntimeBanner';
+
+const ConnectedApps = () => {
+  useEffect(() => {
+    focusElement('[data-focus-target]');
+  }, []);
+
+  return (
+    <>
+      <h2
+        tabIndex="-1"
+        className="vads-u-line-height--1  vads-u-margin-y--2 medium-screen:vads-u-margin-y--4"
+        data-focus-target
+      >
+        Connected Apps
+      </h2>
+      <DowntimeNotification
+        render={handleDowntimeForSection('connected apps')}
+        dependencies={[externalServices.evss]}
+      >
+        <ConnectedApplications />
```

Comment:
I have been following a convention of the main wrapper component rendering a "content" component (see https://github.com/department-of-veterans-affairs/vets-website/blob/f3eafde8be35f4f095c94737df1c1f9f6fffd79c/src/applications/personalization/profile-2/components/AccountSecurity.jsx#L33 and https://github.com/department-of-veterans-affairs/vets-website/blob/f3eafde8be35f4f095c94737df1c1f9f6fffd79c/src/applications/personalization/profile-2/components/direct-deposit/DirectDeposit.jsx#L31, for example. (note: [MilitaryInfo](https://github.com/department-of-veterans-affairs/vets-website/blob/f3eafde8be35f4f095c94737df1c1f9f6fffd79c/src/applications/personalization/profile-2/components/MilitaryInformation.jsx#L1) does not follow this convention; it just has everything in a giant component which I'm not crazy about) I'm not 100% in love with it. But having two `ConnectedApps` components and needing to renaming one on import isn't ideal either.

Writing this note makes me think it would be better to have a `<AppName>Wrapper` component wrap an `<AppName>` component instead of what I've been doing. In other words rename the `DirectDeposit` component to `DirectDepositWrapper` and the `DirectDepositContent` to `DirectDeposit`. I dunno. 🤷

Reply:
I was able to remove the wrapper - I don't think there was a need for it anymore, especially since you mentioned that the `DowntimeNotification` would not be relevant here